inc/views/page-settings.php

```diff
@@ -14,16 +14,23 @@
 
 		<form action="<?php echo esc_url( $settings->get_form_action() ); ?>" id="imagify-settings" method="post">
 
-			<div class="imagify-settings-main-content">
+			<div class="imagify-settings-main-content<?php echo imagify_valid_key() ? '' : ' imagify-no-api-key'; ?>">
 
 				<?php settings_fields( $settings->get_settings_group() ); ?>
 				<?php wp_nonce_field( 'imagify-signup', 'imagifysignupnonce', false ); ?>
 				<?php wp_nonce_field( 'imagify-check-api-key', 'imagifycheckapikeynonce', false ); ?>
 
+				<?php if ( ! imagify_valid_key() ) { ?>
```

Comment:
Not printing the settings fields will simply erase all previously saved settings.